src/main/java/lotto/view/Output.java

```diff
@@ -27,13 +27,22 @@ public static void printLottoResult(final WinningStatistic winningStatistic) {
         printGradeResult(Grade.FOUR_GRADE, winningStatistic);
         printGradeResult(Grade.THIRD_GRADE, winningStatistic);
         printGradeResult(Grade.SECOND_GRADE, winningStatistic);
+        printGradeResult(Grade.BONUS_GRADE, winningStatistic);
         printGradeResult(Grade.FIRST_GRADE, winningStatistic);
         System.out.println(
             MessageFormat.format("총 수익률은 {0}입니다.", winningStatistic.calculateProfitRate()));
     }
 
     private static void printGradeResult(final Grade grade,
         final WinningStatistic winningStatistic) {
+        if (grade == Grade.BONUS_GRADE) {
```

Comment:
메시지를 보내서 처리해보면 어떨까요?
```suggestion
        if (grade.isBonusGrade()) {
```

Reply:
이 부분 피드백 반영해놓겠습니다!